packages/design-system-react-native/src/primitives/ButtonBase/ButtonBase.constants.ts

```diff
@@ -0,0 +1,33 @@
+import { IconSize, IconColor } from '../../components/Icon';
+import { TextColor, TextVariant, FontWeight } from '../../components/Text';
+import type { ButtonBaseProps } from './ButtonBase.types';
+import { ButtonBaseSize } from './ButtonBase.types';
+
+// Defaults
+export const DEFAULT_BUTTONBASE_PROPS: Partial<ButtonBaseProps> = {
+  textProps: {
+    variant: TextVariant.BodyMd,
+    fontWeight: FontWeight.Medium,
+    color: TextColor.TextDefault,
+    numberOfLines: 1,
+    ellipsizeMode: 'clip',
+  },
+  size: ButtonBaseSize.Lg,
+  isLoading: false,
+  spinnerProps: {
+    color: IconColor.IconDefault,
+  },
+  startIconProps: {
+    size: IconSize.Sm,
+    testID: 'start-icon',
+  },
+  endIconProps: {
+    size: IconSize.Sm,
+    testID: 'end-icon',
+  },
+  onPress: () => {
+    console.log('Button pressed');
```

Comment:
non-blocking: We shouldn't use a console.log as default props. You could consider using a no-op function (`() => {}`) or removing the default `onPress` handler entirely.